episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,847 @@
+  mu = mers_offspring["mean"],
+  size = mers_offspring["dispersion"],
+  # generation
+  generation_time = function(x) generate(x = serial_inverval, times = x)
+)
+```
+
+`simulate_chains()` requires at least three sets of arguments:
+
+- simulation controls,
+- offspring distribution, and
+- generation time.
+
+In the lines above, we described how to specify the offspring distribution and generation time. The **simulation controls** include at least two arguments:
+
+- `index_case`, which defines the number of index cases to simulate transmission chains for and
+- `statistic`, which defines a chain statistic to track (either `"size"` or `"length"`) as the stopping criteria for each chain being simulated.
+
+::::::::::::::::::::::::::: callout
+
+### Stopping criteria
+
+This is an exclusive feature of `{epichains}`. Branching process simulations end when they have gone extinct. No more offspring are being produced because of some stopping criterion.
+
+:::::::::::::::::::::::::::
+
+The `simulate_chains()` output creates a `<epichains>` class object.
+
+## Simulate multiple chains
+
+We can use `simulate_chains()` to create multiple chains and increase the probability of simulating uncontrolled outbreak projections given an overdispersed offspring distribution.
+
+We need to specify three additional elements:
+
+- `set.seed(<integer>)`, which is a random number generator function with a specified seed value, the `<integer>` number, to ensure consistent results across different runs of the code.
+- `number_chains`, which defines the number of simulations to run.
+- `initial_cases` defines the number of initial cases to input to the `index_cases` argument explained in the lines above.
+
+```{r}
+# Set seed for random number generator
+set.seed(33)
+# Number of simulation runs
+number_chains <- 1000
+# Number of initial cases
+initial_cases <- 1
+```
+
+`number_chains` and `initial_cases` are conveniently stored in objects to facilitate downstream reuse in the workflow.
+
+:::::::::::::::::::::::::::::: checklist
+
+### Iteration using purrr
+
+[Iteration](https://r4ds.hadley.nz/iteration.html) aims to perform the same action on different objects repeatedly.
+
+Learn how to use the core `{purrr}` functions like `map()` from the YouTube tutorial on [How to purrr](https://www.youtube.com/watch?v=nd-Y8b22YaQ) by Equitable Equations.
+
+Or, if you previously used the `*apply` family of functions, visit the package vignette on [purrr base R](https://purrr.tidyverse.org/articles/base.html), which shares key differences, direct translations, and examples.
+
+::::::::::::::::::::::::::::::
+
+To get multiple chains, we must apply the `simulate_chains()` function to each chain defined by a sequence of numbers from 1 to `number_chains`.
+
+::::::::::::::::::::::::::::::: callout
+
+### purrr and epichains
+
+First, let's sketch how we use `purrr::map()` with `epichains::simulate_chains()`. The `map()` function requires two arguments:
```

Comment:
Worth mentioning (or reminding) that `::` is just calling a function from a specific package?

Reply:
testing to have this right after loading packages in all episodes https://github.com/epiverse-trace/tutorials-middle/commit/b00d13bb28c205086093e68a8a7a52d49bb11d8d